### PR TITLE
Mjb 119584003 query if defaults have not changed

### PR DESCRIPTION
**Pull request by cbitstech**
[#119584003]

### CODE REVIEW

File: src/js/controllers/EmaController.js [hunk 1]
```diff
@@ -26,23 +27,53 @@
...
-      dislike: 5
+      dislike: 5,
+      changedFromDefault: false
     };
 
+    this.updateChangeFromDefault = function(model){
+      model.changedFromDefault = true;
+    }
+
+    this.openEmaConfirmationModal = function () {
```
Comment (ericcf) [inline src/js/controllers/EmaController.js]: Is there a reason not to put this logic in a directive?

File: src/js/controllers/EmaController.js [hunk 2]
```diff
@@ -26,23 +27,53 @@
...
+        templateUrl: 'partials/ema_confirmation_modal.html',
+        controller: function($scope, $modalInstance){
+          $scope.close = function() {
+            $modalInstance.close();
+          };
+        }
+      })
+    };
+
+    this.proceed = function(state,model){
```
Comment (ericcf) [inline src/js/controllers/EmaController.js]: Is there a reason not to write a unit test for this method?

Comment (cbitstech) [inline src/js/controllers/EmaController.js]: adding!

File: src/js/controllers/EmaController.js (hunk 1 above)
Comment (cbitstech) [inline src/js/controllers/EmaController.js]: for the moment it is to retain consistence with the existing codebase--but agreed moving forward

**Pull request merged**